charts/keycloakx/values.yaml

```diff
@@ -236,7 +236,7 @@ service:
   # The HTTP Service node port if type is NodePort
   httpNodePort: null
   # The HTTPS Service port
-  httpsPort: 8443
+  httpsPort: null
```

Comment:
Could you please revert the changes to the defaults? I agree to disable the https as a default would make sense. But, i do not have the time to "fix" the release-automation, so in order to merge this change sooner, i'd suggest to stick to the default for now.

Reply:
done